float/src/main/java/com/novoda/floatschedule/convert/GithubUserConverter.java

```diff
@@ -0,0 +1,13 @@
+package com.novoda.floatschedule.convert;
+
+import java.util.List;
+
+public interface GithubUserConverter {
+    
+    List<String> getGithubUsers() throws Exception;
```

Comment:
in the end i went with a middle-ground approach,  wrapping up the `IOException` in a new one: `FailedToLoadMappingsException`. this way the exception still provides meaning and info on why something went wrong.